service/configservice/api.go

```diff
@@ -4029,6 +4029,7 @@ func (s *DeliveryChannel) SetSnsTopicARN(v string) *DeliveryChannel {
 // The status of a specified delivery channel.
 //
 // Valid values: Success | Failure
+//
```

Comment:
@jasdel let me know when the api.go fixes have been sent upstream, and then I will undo the changes from the PR.